README.md

```diff
@@ -1,63 +1,29 @@
-# epitools
+# EpiTools
 
-[![License](https://img.shields.io/pypi/l/epitools.svg?color=green)](https://raw.githubusercontent.com/epitools/epitools/main/LICENSE)
+[![Licence](https://img.shields.io/pypi/l/epitools.svg?color=green)](https://raw.githubusercontent.com/epitools/epitools/main/LICENCE.md)
 [![PyPI](https://img.shields.io/pypi/v/epitools.svg?color=green)](https://pypi.org/project/epitools)
 [![Python Version](https://img.shields.io/pypi/pyversions/epitools.svg?color=green)](https://python.org)
 [![tests](https://github.com/epitools/epitools/actions/workflows/test.yml/badge.svg)](https://github.com/epitools/epitools/actions/workflows/test.yml)
 [![coverage](https://coveralls.io/repos/github/epitools/epitools/badge.svg?branch=main)](https://coveralls.io/github/epitools/epitools?branch=main)
 [![napari hub](https://img.shields.io/endpoint?url=https://api.napari-hub.org/shields/epitools)](https://napari-hub.org/plugins/epitools)
 
-EpiTools Plugin
-
----
-
-This [napari] plugin was generated with [Cookiecutter] using [@napari]'s [cookiecutter-napari-plugin] template.
-
-<!--
-Don't miss the full getting started guide to set up your new package:
-https://github.com/napari/cookiecutter-napari-plugin#getting-started
-
-and review the napari docs for plugin developers:
-https://napari.org/plugins/index.html
--->
-
 ## Installation
 
-`epitools` is not yet available to install from PyPI or napari-hub.
-
-To install, please clone the repository and do a local install using `pip`:
+The recommended way to install `EpiTools` is via
+[pip](https://pypi.org/project/pip)
 
+```sh
+pip install epitools
```

Comment:
I think:

```
python -m pip install epitools
```

should be the recommended way to install it, in line with the official [Python](https://docs.python.org/3/installing/index.html#basic-usage) and [PyPA](https://pip.pypa.io/en/stable/cli/pip_install/) docs

Reply:
Yep sure